.gitpod.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM gitpod/workspace-mysql
+
+USER root
+
+# Update APT Database
+### base ###
+RUN sudo apt-get update -q \
```

Comment:
Since `root` is the current user, do we really need `sudo` here?

Reply:
its true, fixed now